website/content/docs/concepts/api-lock.mdx

```diff
@@ -0,0 +1,82 @@
+---
+layout: docs
+page_title: Namespace API Lock
+description: Lock the Vault API on a per-namespace basis.
+---
+
+# Lock/Unlock
+
+Vault makes the API available (unlocked) by default for all namespaces. In
+this state, Vault can respond to all API/CLI ('API' from here on out) requests
+as it usually would.
+
+A Vault Administrator can lock the API for particular namespaces. In this state,
+Vault will block all but a select few API endpoints from responding to clients
+operating in a locked namespace (or a descendant of a locked namespace). The
+endpoints that do respond, the exempt paths, are largely for checking the status
```

Comment:
Would it be possible to enumerate the exempt paths, or if there are too many to list here maybe a few that an operator would be most likely to use in a locked state?

Reply:
b3ba09640 and 331f67544